### PR TITLE
Feat: Support both askcode and askscript in askExpressMiddleware

### LINKED ISSUE #191: demoServer.ts should accept AskScript instead of AskCode
The variable askScriptCode in https://github.com/xFAANG/askql/blame/master/src/askExpressMiddleware/askExpressMiddleware.ts#L20 suggests AskScript should be accepted but actually the server throws syntax errors for AskScript programs, while accepting AskCode.

### PR DESCRIPTION
**Pull request by YonatanKra**
Closes #191

### CODE REVIEW

File: src/askExpressMiddleware/askExpressMiddleware.ts [hunk 1]
```diff
@@ -16,8 +16,8 @@ export function askExpressMiddleware(
     next: NextFunction
   ) {
     try {
-      const askScriptCode = request.body.code;
-      const queryResponse = await runUntyped(environment, parse(askScriptCode));
+      const askCode = request.body.code;
+      const queryResponse = await runUntyped(environment, parse(askCode));
```
Comment (mhagmajer) [inline src/askExpressMiddleware/askExpressMiddleware.ts:20]: `parse` should come from the askscript package now, not askcode :)

Comment (mhagmajer) [reply src/askExpressMiddleware/askExpressMiddleware.ts:20]: We'll support both AskCode and AskScript inputs (even without requiring user to provide an option) , but for now we need AskScript support for playground

**Comment by mhagmajer** [comment]
Also please update the PR name according to conventional commits :)

File: src/askExpressMiddleware/askExpressMiddleware.ts (hunk 1 above)
Comment (YonatanKra) [reply src/askExpressMiddleware/askExpressMiddleware.ts]: @mhagmajer 
Please have a look now.
Also please look at the new test case. Is this a valid test for askscript?

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,5 @@
 node_modules
-
+/.idea
```
Comment (mhagmajer) [inline .gitignore:2]: .globalignore

Comment (mhagmajer) [reply .gitignore:2]: See https://github.com/xFAANG/askql/issues/120 for reference

Comment (YonatanKra) [reply .gitignore:2]: Cool @mhagmajer ! Didn't know that :)
Solves many things in my many projects :)
So the code's working?

**Pull request merged**